lentils_tacos.md

```diff
@@ -0,0 +1,6 @@
+Using the same spices we can use cooked lentils instead of meat.
+*cooked lentils
+*different spices
+*mix and warm
+*vegetables
+*taco things
```

Comment:
I wonder what the best taco things could be? We could maybe make a list of them in this file or a separate file?

Reply:
Great point @jaimemcc 🙌
Maybe we can add a separate file and link it to this file? Whatever feels convenient.

Reply:
probably some frijoles (beans) could be a good addition

Reply:
Yes. I agree I send a new one